### ISSUE TITLE
Security issue [ bug ] [ REFLECTED XSS]

### ISSUE THREAD

**Issue by n0taSkid** [FortAwesome/Font-Awesome#18140]
**Describe the bug**
Hello,
I started using font-awesome for my forum when I saw a link with an interesting parameter...I'm not sure if I can disclose the bug here by showing where it is and how to reproduce it so that's why now I want to ask this ... Am I able to disclose here the bug?

**To Reproduce**
Still not 

**Screenshots**
Still not 

**Bug report checklist**

**Comment by n0taSkid** [comment]
Shoud I report it somewhere else?

**Comment by tagliala** [comment]
@n0taSkid yes, please send an email at hello@fontawesome.com to the attention of Rob @robmadole

**Comment by n0taSkid** [comment]
oK

**Comment by n0taSkid** [comment]
I sended an email with the exact same message as I wrote above...or shoud I directly send the report? How can I write that email so robmadole will see...?

**Comment by tagliala** [comment]
Hi, please give the team some time to answer. Yesterday we have released FA 5.13.4

Let me know in 1 day if someone got in touch

**Comment by n0taSkid** [comment]
Ok no problem

**Comment by robmadole** [comment]
@n0taSkid why don't you send it directly to me please. It doesn't seem to be in our team inbox.

rob@fontawesome.com

**Comment by n0taSkid** [comment]
[ REFLECTED XSS + OPEN REDIRECT ] THE REPORTHello, I’m so sorry , I didn’t know this was your email .So few days ago when I was searching for icons for my forum,I found Font Awsome .. so I tryed log myself into [ like I made an account] and surely I started using everything ...but when I got loged out somehow, I started searching for a sign in button for loging myself back to font awesome ,this was the button I clicked for receiving the vulnerable parameter  : [the photo is screenshot_5 ] I got a link looking like this : [the photo is screenshot_5 ] https://fontawesome.com/sessions/sign-in?next=%2Fstart ,now I started thinking what if I can change the value of the  parameter ?next= for creating an interesting open redirect after log...so I change the ?next= with https://google.com/ and when user as an example is clicking the link https://fontawesome.com/sessions/sign-in?next=https://google.com/ he/she will get redirected to the page where he/she shoud login and after he/she clicks the log in button the user will get instantly redirected to the value of the parameter ?next which in my case is https://google.com/ . What about reflected xss?  Everytime when I see a open redirect, I always try to escalate it to a reflected xss which in my case worked amazing ,the stepts are more than simple : just change the ?next= value with a simple javascript payload that shoud look like this : javascript:alert() and after the user introduces the password for the account and the email ,the javascript code will get executed!  Here is an simple example : https://fontawesome.com/sessions/sign-in?next=javascript:alert()   Kind regards , wating for your response!

**Comment by n0taSkid** [comment]
Hey, I sended you the report, sorry I didn;t know your email ....

**Comment by n0taSkid** [comment]
I think m
y first email was market as spam

**Comment by robmadole** [comment]
Got the report. Thank you @n0taSkid

**Comment by n0taSkid** [comment]
Hello,
I forgot to ask some questions about this , so here are all: 
Will I receive any update from fixing ? Did anybody from the security team saw this ? Bug hunters that found something in font awesome are receiving any rewards?
Thanks , wating for response!

**Comment by n0taSkid** [comment]
Someone here?

**Comment by tagliala** [comment]
Hi @n0taSkid, Rob is based in the US, will ask to him if he received the email later this afternoon

**Comment by n0taSkid** [comment]
Oh ,ok

**Comment by n0taSkid** [comment]
Hey so right now I'm still wating is anything alright with the report?

**Comment by robmadole** [comment]
@n0taSkid thanks again for the report. You'll need to give us a little time to work through it though.

**Comment by n0taSkid** [comment]
Oh ok sorry 
But about the questions above can I get a response?

**Comment by n0taSkid** [comment]
These I mean : I forgot to ask some questions about this , so here are all:
Will I receive any update from fixing ? Did anybody from the security team saw this ? Bug hunters that found something in font awesome are receiving any rewards?

**Comment by robmadole** [comment]
We'll answer your questions when we've had the time to evaluate your report.

**Comment by n0taSkid** [comment]
Ok

**Comment by n0taSkid** [comment]
Hi , sorry for asking but how much time it usually takes for this type of reports to get verified?

**Comment by n0taSkid** [comment]
Hi , sorry for asking but how much time it usually takes for this type of reports to get verified?

**Comment by attritionorg** [comment]
Harassing the maintainers of an open source project isn't a way to encourage them to process your report quickly. Pretty sure they don't get paid to do this.

**Comment by tagliala** [comment]
@attritionorg thanks for your willing to support. I've removed your comment to prevent off topics

Font Awesome is also a commercial product. I do not have news about this report

**Comment by n0taSkid** [comment]
I think you responded to the wrong thread....who is attritionorg? What it has to do with my report?

**Comment by n0taSkid** [comment]
Hello, 
I'm waiting for a week now ... can I know when exactly this report will get solved?

**Comment by robmadole** [comment]
@n0taSkid you'll have to be patient. We'll look at this as soon as we can.

**Comment by n0taSkid** [comment]
Oh ok
